README.md

```diff
@@ -15,9 +15,9 @@
 In order to access data from contentful make sure that you have an .env file that matches the contentful keys and tokens:
 
 ```
-CONTENTFUL_TOKEN=(see in contentful/settings/API keys)
-CONTENTFUL_SPACE=(see in contentful/settings/API keys)
-MEETUP_KEY=(see in contentful/settings/API keys)
+CONTENTFUL_TOKEN=(see in contentful/settings/API keys "meetup > Content Delivery API - Access Token")
```

Comment:
I don't understand the `meetup` keyword in a Contentful-related env var. But probably I'm missing something :-)

Reply:
Are we talking about the `MEETUP_KEY` itself or the descriptions I've added?

If it's the descriptions, there are two spaces in contentful and each one has two sets of API keys. The API Keys in `yld.io` space are `meetup` and `gatsby`

Reply:
ok se we've got meetup-related keys inside contentful? interesting :-)